core/field_checkbox.js

```diff
@@ -116,6 +131,19 @@ Blockly.FieldCheckbox.prototype.CURSOR = 'default';
  */
 Blockly.FieldCheckbox.prototype.isDirty_ = false;
 
+/**
+ * Configure the field based on the given map of options.
+ * @param {Object} opt_config A map of options to configure the field based on.
+ * @private
+ */
+Blockly.FieldCheckbox.prototype.configure_ = function(opt_config) {
+  if (opt_config) {
+    if (opt_config['checkCharacter']) {
+      this.checkChar_ = opt_config['checkCharacter'];
```

Comment:
Okay, one last thing. Can we call setCheckCharacter here so it all leads to the same code path. This method becomes a property bag to call a bunch of configuration methods. ie: 
```js
new FieldCheckbox(val, null, {
   checkCharacter: '\u2713'
});
```
is shorthand for having done: 
```js
var field = new FieldCheckbox(val);
field.setCheckCharacter('\u2713');
```